### PR TITLE
Fix last column with relative/absolute flags

### PR DESCRIPTION
**Pull request by christian-forgacs**
This is:

- [x] a bugfix

Checklist:

- [x] Changes are covered by unit tests
- [x] Code style is respected
- [x] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [x] CHANGELOG.md contains a short summary of the change
- [-] Documentation is updated as necessary

### Why this change is needed?

If you read a xls file and in this file you use an absolut reference in SUM function e.g. =SUM(SHEET!A1:SHEET!B1) the script only returns =SUM(SHEET!A1:SHEET!A1)

### CODE REVIEW

**Comment by christian-forgacs** [comment]
@PowerKiKi the unit tests were not broken by my changes. Currently your develop branch is broken (2 errors from unit tests). I add some unit tests for this bugfix and push it this weekend.

**Comment by christian-forgacs** [comment]
@PowerKiKi can you explain in witch case the readBIFF8CellRangeAddressB function is used? In my regular project xls file this function is called by an cell with sum function with a range reference.

I reconstruct the case from this project xls file but always readBIFF8CellRangeAddress is called.

What is the different between tArea and tAreaN? With this information I can create a unit test with an xls file because I can publish the xls from my project.

**Comment by christian-forgacs** [comment]
@PowerKiKi I add the unit test for this bugfix and fix the other unit tests with a update for dompdf (from 0.8.0 to 0.8.2).

File: tests/PhpSpreadsheetTests/Reader/XlsTest.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+            foreach ($testsForSheet as $cellCoordinate => $result) {
+                $calculatedValue = $sheet->getCell($cellCoordinate)->getCalculatedValue();
+
+                if ($calculatedValue != $result) {
+                    if (empty($calculatedValue) === true) {
+                        $calculatedValue = 'empty';
+                    }
+
+                    throw new \Exception($cellCoordinate . ' != ' . $result . ' - Result in sheet with name ' . $sheetName . ' is ' . $calculatedValue . '.');
```
Comment (PowerKiKi) [inline tests/PhpSpreadsheetTests/Reader/XlsTest.php]: This should be a proper assertion such as `self::assertSame`

File: tests/PhpSpreadsheetTests/Reader/XlsTest.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+use PHPUnit\Framework\TestCase;
+
+class XlsTest extends TestCase
+{
+    /**
+     * Test load Xls file without cell reference.
+     */
+    public function testLoadXlsWithoutCellReference()
+    {
+        $filename = './data/Reader/Xls/without_cell_reference.xls';
```
Comment (PowerKiKi) [inline tests/PhpSpreadsheetTests/Reader/XlsTest.php]: We strive to limit the number of binary files in the repository to keep its size down. Can you replace those with by writing them with PhpSpreadsheet and then reloading them instead ? (this is why I suggested functional tests, see `\PhpOffice\PhpSpreadsheetTests\Functional\AbstractFunctional`)

**Comment by oleibman** [comment]
Superseded by PR #4140.

**Pull request closed without merging**